versioned/transfer/src/test/java/org/projectnessie/versioned/transfer/TestContentMigration.java

```diff
@@ -91,7 +87,7 @@ private void importRepo() throws IOException {
   private ExportMeta exportRepo(String branchName, int batchSize) throws IOException {
     NessieExporter exporter =
         NessieExporter.builder()
-            .databaseAdapter(databaseAdapter)
+            .persist(srcPersist)
```

Comment:
Why keep this test? Do we not cover `Persist`-based export / import in other tests?

Reply:
Yea - this test class can go away - committed